robotiq_s_model_articulated_msgs/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(robotiq_s_model_articulated_msgs)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS std_msgs message_generation )
+
+add_message_files(
+    FILES
+    SModelRobotInput.msg 
+    SModelRobotOutput.msg
+) 
+
+generate_messages(
+    DEPENDENCIES
+    std_msgs
+)
+
+catkin_package(
+    DEPENDS std_msgs
+)
+
+
+install(DIRECTORY msg/  DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION})
```

Comment:
Ah,meant to go to the catkin package share destination.
On Jan 18, 2015 6:21 AM, "G.A. vd. Hoorn" <notifications@github.com> wrote:

> In robotiq_s_model_articulated_msgs/CMakeLists.txt
> <https://github.com/ros-industrial/robotiq/pull/17#discussion-diff-23131689>
> :
>
> > +    FILES
> > +    SModelRobotInput.msg
> > +    SModelRobotOutput.msg
> > +)
> > +
> > +generate_messages(
> > +    DEPENDENCIES
> > +    std_msgs
> > +)
> > +
> > +catkin_package(
> > +    DEPENDS std_msgs
> > +)
> > +
> > +
> > +install(DIRECTORY msg/  DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION})
>
> Are msgs ever installed this way? A quick look at pkgs in common_msgs
> shows this to not be the case.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ros-industrial/robotiq/pull/17/files#r23131689>.
>